docker-compose.yml

```diff
@@ -811,6 +811,57 @@ services:
         max-size: 20m
         max-file: '5'
 
+  http-agent-cert-sidecar:
+    image: dojot/cert-sidecar:${DOJOT_VERSION}
+    depends_on:
+      - x509-identity-mgmt
+    environment:
+      CERT_SC_APP_SIDECAR_TO: 'http-agent'
+      CERT_SC_LOG_CONSOLE_LEVEL: 'info'
+      CERT_SC_CERTS_HOSTNAMES: '["http-agent", "localhost"]'
+      HOSTNAME: http-agent
+      CERT_SC_CERTS_FILES_BASEPATH: /certs
+      CERT_SC_CERTS_FILES_CA: ca.crt
+      CERT_SC_CERTS_FILES_CERT: http-agent.crt
+      CERT_SC_CERTS_FILES_KEY: http-agent.key
+      CERT_SC_CERTS_FILES_CRL: ca.crl
+      CERT_SC_CRON_CRL_TIME: '0 */1 * * * *'
+    volumes:
+      - http-agent-cert-side-volume:/certs
```

Comment:
Just an aesthetic comment... I think the volume name could follow the "dojot pattern", so for this you need to change `http-agent-cert-side-volume` to `http-agent-cert-sidecar-volume`.